lib/letter_opener/tasks/letter_opener.rake

```diff
@@ -0,0 +1,9 @@
+require 'rails/tasks'
+
+namespace :tmp do
+  task :letter_opener do
+    rm_rf Dir["tmp/letter_opener/[^.]*"], verbose: false
```

Comment:
can you please use `LetterOpener.configuration.location` here so it works for users who use another location as well?

Reply:
Done